pom.xml

```diff
@@ -632,6 +633,18 @@
             <artifactId>mockito-inline</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why add powermock?

Reply:
```java
        clientEventExecutor = new ScheduledThreadPoolExecutor(2, r -> {
            Thread t = new Thread(r);
            t.setName("com.alibaba.nacos.client.remote.worker");
            t.setDaemon(true);
            return t;
        });
```

```ScheduledThreadPoolExecutor::new``` cannot be mock by mockito, so i use PowerMock to mock the return value of ```ScheduledThreadPoolExecutor::new```.

Reply:
use reflection to get clientEventExecutor and replace it with mock `ScheduledThreadPoolExecutor`

Reply:
powermock will change the compiled file may cause some unexpected problem.

Reply:
In the test case, I can mock the ```clientEventExecutor``` and assign it to the ```clientEventExecutor``` through reflection. However, in the run of ```Rpcclient:: start``` method, it will reassign a value to the ```clientEventExecutor``` which will overwrite the ```clientEventExecutor``` of the previous mock.

The testability of this method is low. I didn't think of a better way to test it. Should i delete all test cases related to this method? Or put the initialization of ```clientEventExecutor``` out of the method for testability?